ipatests/test_xmlrpc/test_certmap_plugin.py

```diff
@@ -122,6 +139,23 @@ def test_delete(self, certmap_rule):
         certmap_rule.ensure_exists()
         certmap_rule.delete()
 
+    @pytest.mark.parametrize(
+        'dont_fill',
+        itertools.chain(*[
+            itertools.combinations(certmaprule_optional_params, l)
+            for l in range(len(certmaprule_optional_params) + 1)
+        ]),
+        ids=dontfill_idfn,
+    )
+    def test_failed_create(self, dont_fill, certmap_rule_trusted_domain):
+        certmap_rule_trusted_domain.ensure_missing()
+        try:
+            certmap_rule_trusted_domain.create(dont_fill)
+        except errors.ValidationError:
+            pass
+        finally:
```

Comment:
The try/except clause is missing a else: statement. When the certmaprule-create does not raise the expected ValidationError, the test should exit on error.

Reply:
Thanks. I added raising an assertion error for this case.